draft-bbs-signatures.md

```diff
@@ -637,6 +664,9 @@ point\_to\_octets
 octets\_to\_point
 : follows the format documented in Appendix C section 2 of [@!I-D.irtf-cfrg-pairing-friendly-curves].
 
+dst
+: "BBS_BLS12381G1_XOF:SHAKE-256_SSWU_RO"
```

Comment:
This should have an extra underscore on the end

Reply:
@andrewwhitehead can you explain why?

Reply:
In the code it does :)

Reply:
In the hash to curve spec they leave an underscore at the end as well. Its mostly for consistency.

Reply:
Have updated this now, thanks